packages/gatsby-source-contentful/.gitignore

```
@@ -1,4 +1,33 @@
-/__tests__
-/yarn.lock
-/*.js
+# Logs
+logs
+*.log
+
+# Runtime data
+pids
+*.pid
+*.seed
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directory
+# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
+node_modules
+
+decls
+dist
+*.js
+!gatsby-node.js
```

`.babelrc.js` and `jest.config.js` are covered by gitignore currently which trips some scripts (that is cleaning package dirs from stale built files - it also removes those files that later need to be manually restored)
```suggestion
!gatsby-node.js
!.babelrc.js
!jest.config.js
```